snuba/migrations/migration.py

```diff
@@ -45,13 +46,64 @@ def backwards(self, context: Context, dry_run: bool) -> None:
         raise NotImplementedError
 
 
-class MultiStepMigration(Migration, ABC):
+class GlobalMigration(Migration, ABC):
```

Comment:
Maybe `CodeMigration`? To me this class name implies something that can have SQL and Python migrations.

Reply:
`CodeMigration` makes sense to me - updated